langchain/agents/agent.py

```diff
@@ -913,17 +1074,30 @@ def _call(
             [tool.name for tool in self.tools], excluded_colors=["green"]
         )
         intermediate_steps: List[Tuple[AgentAction, str]] = []
+        if self.reflector:
+            self.reflector.reset()
         # Let's start tracking the number of iterations and time elapsed
-        iterations = 0
-        time_elapsed = 0.0
-        start_time = time.time()
+        # for total execution, and for current trial
+        total_iterations = 0
+        total_time_elapsed = 0.0
+        total_start_time = time.time()
+        trials = 1
+        trial_iterations = 0
```

Comment:
trials and trial_iterations is confusing to me. From my understanding trial_iterations indicates the number of iterations done during the current trial, and "trials" indicates the current trial number. Maybe it would be beneficial to add some brief comment to describe each arguments or find a better naming (I don't have ideas thought).

Reply:
Good comment - will think of this when I get further comments from the LC team